### PR TITLE
fix(bugbash): bugs in compression and speedup

### PR DESCRIPTION
**Pull request by Louis-J**
### Description ###

1. fix the bug in compression that dependency_aware do not support functions like torch.div, torch.sub
2. fix the bug in speedup of support of aten::ones_like, zeros_like by adding a convertion from 'int layout' in TorchScript to 'torch.layout' in python.

#### Test Options ####
  - [ ] fast test
  - [ ] full test - HPO
  - [ ] full test - NAS
  - [x] full test - compression

### Checklist ###
  - [x] test case
  - [x] doc

### CODE REVIEW

File: docs/source/compression/pruning.rst [hunk 1]
```diff
@@ -105,4 +105,5 @@ In addition, for the convolutional layers that have more than one filter group,
 ``dependency-aware pruner`` will also try to prune the same number of the channels for each filter group.
 Overall, this pruner will prune the model according to the L1 norm of each filter and try to meet the topological constrains (channel dependency, etc) to improve the final speed gain after the speedup process. 
 
+Operations that will be recognized as having channel dependencies: add/sub/mul/div, addcmul/addcdiv, logical_and/or/xor
```
Comment (J-shang) [inline docs/source/compression/pruning.rst:108]: .. Note:: Operations that will be recognized as having channel dependencies: add/sub/mul/div, addcmul/addcdiv, logical_and/or/xor.

**Review by J-shang — APPROVED** [review]

**Review by zheng-ningxin — APPROVED** [review]
LGTM

**Pull request merged**